src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Backup/GetAzureRmRecoveryServicesBackupStatus.cs

```diff
@@ -56,42 +59,61 @@ public class GetAzureRmRecoveryServicesBackupStatus : RecoveryServicesBackupCmdl
         /// </summary>
         [Parameter(ParameterSetName = NameParamSet, Mandatory = true,
             HelpMessage = ParamHelpMsgs.ProtectionCheck.Type)]
+        [Parameter(ParameterSetName = IdWorkloadParamSet, Mandatory = true,
+            HelpMessage = ParamHelpMsgs.ProtectionCheck.Type),]
         [ValidateSet("AzureVM", "AzureFiles")]
         public string Type { get; set; }
 
         [Parameter(ParameterSetName = IdParamSet, ValueFromPipelineByPropertyName = true,
             HelpMessage = ParamHelpMsgs.ProtectionCheck.ResourceId, Mandatory = true)]
+        [Parameter(ParameterSetName = IdWorkloadParamSet, ValueFromPipelineByPropertyName = true,
+            HelpMessage = ParamHelpMsgs.ProtectionCheck.ResourceId, Mandatory = true)]
         public string ResourceId { get; set; }
 
+
+        [Parameter(ParameterSetName = IdWorkloadParamSet, ValueFromPipelineByPropertyName = true,
+          HelpMessage = ParamHelpMsgs.ProtectionCheck.ProtectableObjName, Mandatory = true)]
+        public string ProtectableObjName { get; set; }
```

Comment:
Use ProtectabeObjectName - You can have a parameter alias to shorter names if you like.

Reply:
Done